### PR TITLE
[Docs] External Network

### PR DESCRIPTION
**Pull request by aleoli**
# Description

This pr adds documentation for the external network feature

### CODE REVIEW

File: docs/advanced/use-only-offloading.md [hunk 1]
```diff
@@ -43,3 +42,20 @@ This flag disables the reflection of the IP addresses.
...
+## Disable Networking for a specific Cluster
+
+It is also possible to disable the networking module for a specific cluster.
+This can be done by adding the `--disable-networking` flag to the `liqoctl peer` command when establishing a peering.
+
+```bash
+liqoctl --context=consumer peer --remote-kubeconfig <kubeconfig-provider> --disable-networking
+```
+
+The `--disable-networking` flag will prevent the Liqo network CRDs from being created in the clusters, and the networking module will not be enabled for that specific peering. While the ResourceSlices and VirtualNodes will still be created.
```
Comment (claudiolor) [inline docs/advanced/use-only-offloading.md:55]: I wonder whether it is better being less specific, saying something like:
```suggestion
The `--disable-networking` flag will prevent Liqo from configuring the networking between the clusters for that specific peering. While the resource reflection and offloading is still possible.
```

File: docs/advanced/use-only-offloading.md [hunk 2]
```diff
@@ -43,3 +42,20 @@ This flag disables the reflection of the IP addresses.
...
+
+Alternatively, if you are using the [Manual Peering](/advanced/manual-peering) method, you can disable the networking module simply by skipping the Inter-cluster Network Connectivity step.
+
+In that scenario, the Pod IPs will still be reflected in the remote cluster, but they will not be remapped. You can still connect to the remote pods using an external network tool.
+
+In order to disable the IP reflection, you can add the `--disable-ip-reflection` argument to the respective VirtualNode spec by adding it to the `virtualKubelet.extra.args` field in the `values.yaml` file or by adding a custom `VkOptionTemplate` resource to your consumer cluster.
```
Comment (claudiolor) [inline docs/advanced/use-only-offloading.md:61]:
```suggestion
In order to disable the IP reflection, you can add the `--disable-ip-reflection` argument to the respective VirtualNode spec by adding it to the `virtualKubelet.extra.args` field in the `values.yaml` file at installation time or by adding a custom `VkOptionTemplate` resource to your consumer cluster.
```

Comment (claudiolor) [inline docs/advanced/use-only-offloading.md:61]: I would also add an example of the commands to run to achieve this result.

**Review by claudiolor — APPROVED** [review]

File: docs/advanced/use-only-offloading.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    --disable-networking
+```
+
+The `--disable-networking` flag will prevent Liqo from configuring the networking between the clusters for that specific peering. While the resource reflection and offloading is still possible.
+
+Alternatively, if you are using the [Manual Peering](/advanced/manual-peering) method, you can disable the networking module simply by skipping the Inter-cluster Network Connectivity step.
+
+In that scenario, the Pod IPs will still be reflected in the remote cluster, but they will not be remapped. You can still connect to the remote pods using an external network tool.
+
+In order to disable the IP reflection, you can add the `--disable-ip-reflection` argument to the respective VirtualNode spec by adding it to the `virtualKubelet.extra.args` field in the `values.yaml` file at installation time or by adding a custom `VkOptionTemplate` resource to your consumer cluster.
```
Comment (fra98) [inline docs/advanced/use-only-offloading.md:67]:
```suggestion
In order to disable the IP reflection, you can add the `--disable-ip-reflection` argument to the respective VirtualNode spec by adding it to the `virtualKubelet.extra.args` field in the `values.yaml` file at installation time or by adding a custom `VkOptionsTemplate` resource to your consumer cluster.
```

File: docs/advanced/enable-external-networking.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# Enable external networking
+
+Similarly to the [**Use only offloading**](/advanced/use-only-offloading) feature, Liqo allows you to enable [**resource reflection**](/usage/reflection) and [**namespace offloading**](/usage/namespace-offloading) without the need to use the default Liqo networking module to establish pod-to-pod network connectivity between the clusters.
+
+This is useful when the clusters and the pods running on them are **already connected to the same network**.
+In this case, you may enable resource reflection and namespaces offloading without having to establish another network connection between the clusters.
+
+In that case, the suggested configuration is to disable the Liqo networking module but leave the IP reflection enabled (i.e., `--set networking.enabled=false --set networking.reflectIPs=true`, see [here](AdvancedUseOnlyOffloadingDisableModule), or do not establish network connectivity between the clusters without adding the `--disable-networking` argument to the `VirtualNode` resource, see [here](AdvancedUseOnlyOffloadingSpecificCluster)).
```
Comment (fra98) [inline docs/advanced/enable-external-networking.md:8]: Do you mean the `.spec.disableNetworkCheck` field of the `VirtualNode` resource?

**Review by fra98 — APPROVED** [review]

File: docs/advanced/enable-external-networking.md (hunk 4 above)
Comment (aleoli) [reply docs/advanced/enable-external-networking.md:8]: No, this was an old flag to skip the check for IP resources... probably we had an old base for the branch where the flag was still there

**Pull request merged**